docs/basics/basic_benchmark.md

```diff
@@ -0,0 +1,139 @@
+# Basic Benchmark Structure
+
+
+Every benchmark starts with these global configuration.
+
+```yaml
+name: BENCHMARK_NAME
+http:
+  host: URL
+  sharedConnections: CONNECTIONS
+phases: ...
+```
+
+* URL is the HTTP protocol and hostname of the Application Under Test (e.g. http://myapp.com)
+* Shared connections are physical HTTP (1.x or2) connection.
+
+Example: 
+
+```yaml
+name: My_App
+http:
+  host: http://myapp.com:9080
+  sharedConnections: 10
+phases: ...
+```
+
+## Multiple Core Support
+
+* By default, Hyperfoil only utlize single core for the execution for experimenting & debugging.  For production usage, add the `threads` that matches to the total CPU core numbers.
+
+```yaml
+name: BENCHMARK_NAME
+threads: CPU_CORES
+http:
+  host: URL
+  sharedConnections: CONNECTIONS
+phases: ...
+```
+
+Example: 
+
+```yaml
+name: My_App
+threads: 4
+http:
+  host: http://myapp.com:9080
+  sharedConnections: 10
+phases: ...
+```
+
+## Private HTTP Pools
+
+* To keep connection private to the session until the session completes and prefer these connection, use `ergonomics` with `privateHttpPools` true. (Note: Session is the state of one VU.)
+
+```yaml
+name: BENCHMARK_NAME
+threads: CPU_CORES
+ergonomics:
+  privateHttpPools: true
+http:
+  host: URL
+  sharedConnections: CONNECTIONS
+phases: ...
+```
+
+Example: 
+
+```yaml
+name: My_App
+threads: 4
+ergonomics:
+  privateHttpPools: true
+http:
+  host: http://myapp.com:9080
+  sharedConnections: 10
+phases: ...
+```
+
+## DNS providing multiple IPs for load-balancing
+
+* When DNS is providing multiple IPs for load-balancing, use the `addresses` (e.g. service mesh such as ISTIO) 
```

Comment:
It's not only for load balancing. Maybe we should formulate this as going through one or more proxies? Your example shows single address so it does not really fit to the description.
Maybe use `PROXY_IP:PROXY_PORT` (yes, it takes port as well) and `ANOTHER_PROXY_IP` instead of `SERVICEMESH_GATEWAY`?

Reply:
ok. I don't really understand how it is working (IP & hosts are different is something I haven't seen before).  thank you for comments.